System-Testing-Deployments-and-Upgrades/Control-System-Upgrade/Deployment-Activities/index.rst

```diff
@@ -22,106 +13,87 @@ If changes are necessary to these scripts from work described in the previous se
-            *docker exec ospl-daemon grep "federations" durability.log*
-    #. Startup Minimal Kubernetes Services (:ref:`TTS <Deployment-Activities-TTS-Minimal-K8S-System>`, :ref:`BTS <Deployment-Activities-BTS-Minimal-K8S-System>`)
-        * This uses the ``sync_apps.py`` script found in `https://github.com/lsst-ts/argocd-csc/bin <https://github.com/lsst-ts/argocd-csc/tree/main/bin>`_.
-        * The script is run in the same place that Kubernetes (*kubectl*) interactions are run.
-        * Log into the argocd pod by doing the following:
-            * *python argocd_login.py /path/to/auth_file*
-            * The *auth_file* must contain the appropriate site username/password on separate lines.
-        * *python sync_apps.py -p*
-        * csc-cluster-config, ospl-config and ospl-daemon apps will be synced automatically.
-        * Once the ospl-daemon app is synced, the script will pause.
-        * Check the logs on Argo CD UI to see if daemons are ready.
-        * Type ``go`` and enter to move onto syncing the kafka-producers app.
-        * Script will again pause once the kafka-producers are synced.
-        * Check that all the logs say "Running" at the bottom. M1M3 now has an indicator saying: "Partial producers are all running".
-        * Once all the kafka-producers are verified to be running, stop here but leave the script alive.
-    #. Startup Bare Metal LOVE (:ref:`Summit <Deployment-Activities-Summit-LOVE-Startup>`, :ref:`TTS <Deployment-Activities-TTS-LOVE-Startup>`, :ref:`BTS <Deployment-Activities-BTS-LOVE-Startup>`).
-    #. Continue Kubernetes Deployment
-        * Go back to where your running ``sync_apps.py`` script is and type ``go`` and enter to move onto syncing the obssys apps.
-        * Use the Argo CD UI to verify that the containers are pulling and running.
-        * The script will pause again here. Do not proceed further with the script, but leave it alive.
-    #. The above now represents a minimal system that other system principles can be allowed to start their daemons/CSCs.
-        * Use the site specific Slack channel (:ref:`Summit <Pre-Deployment-Activities-Summit-Slack-Announce>`, :ref:`TTS <Pre-Deployment-Activities-TTS-Slack-Announce>`, :ref:`BTS <Pre-Deployment-Activities-BTS-Slack-Announce>`) to inform the system principles.
-          In particular notify the people doing the camera upgrade that they can proceed to :ref:`Stage 2<camera-install-stage-2>`.
-    #. Startup Rest of Kubernetes Services.
-        * Go back to where your running ``sync_apps.py`` script is and type ``go`` and enter to proceed with syncing the rest of the apps.
-        * The rest of the apps will be synced automatically so no further intervention is necessary.
+    #. Log into the site specific ArgoCD UI to sync the relevant applications:
+       
+       * Start by syncing ``science-platform``.
+       * If told to do so beforehand, sync ``nublado``.
```

Comment:
For cycle upgrades, which is the main focus of the documentation, nublado will _always_ need to be synced.